pandas/core/arrays/datetimelike.py

```diff
@@ -1860,16 +1863,21 @@ def validate_endpoints(closed):
     left_closed = False
     right_closed = False
 
-    if closed is None:
+    if closed == "both" or closed is True or closed is None:
```

Comment:
I guess we should deprecat this. can you separate this case out (for True/False/None) and show a FutureWarning (may have quite a number of deprecating warnings in the tests so need to change tests)